databases.go

```diff
@@ -50,24 +52,44 @@ const (
 	DatabaseStatusBackingUp    DatabaseStatus = "backing_up"
 )
 
+const (
+	DatabasePlatformRDBMSLegacy  DatabasePlatform = "rdbms-legacy"
+	DatabasePlatformRDBMSDefault DatabasePlatform = "rdbms-default"
+)
+
+const (
+	DatabaseMemberTypePrimary  DatabaseMemberType = "primary"
+	DatabaseMemberTypeFailover DatabaseMemberType = "failover"
+)
+
 // A Database is a instance of Linode Managed Databases
 type Database struct {
```

Comment:
`oldest_restore_time`, `port`, `total_disk_size_gb`, `updates`, `used_disk_size_gb` attributes are still missing?

Reply:
Good catch. I will add `oldest_restore_time` but as for the other fields, we already have a separate ticket to add them.